test/behaviour/graql/GraqlSteps.java

```diff
@@ -369,24 +365,44 @@ public AnswerIdentifierGroup(List<Map<String, String>> answerIdentifierTable, Ma
         }
     }
 
-    private boolean matchAnswer(Map<String, String> answerIdentifiers, ConceptMap answer) {
-
-        if (!(answerIdentifiers).keySet().equals(answer.map().keySet())) {
-            return false;
-        }
+//    private boolean matchAnswer(Map<String, String> answerIdentifiers, ConceptMap answer) {
```

Comment:
what is this comment?

Reply:
It's a method used only by Explanation steps, so for consistency with them (& to fix compiler warning), I've commented it out

Reply:
hmm i can't see the context but is it near the steps that use the method? unclear right now its used for explanations. Might not be clear in core either tbh

Reply:
Added a comment to indicate this